Misc/NEWS.d/next/Library/2018-07-04-21-24-58.bpo-34041.aKk8t5.rst

```diff
@@ -0,0 +1,2 @@
+Add the parameter *deterministic* to the
+sqlite3.Connection.create_function().
```

Comment:
It would be better if we can link to the `create_function()` documentation:

```rst
.. untested
:meth:`sqlite3.Connection.create_function`
```

Reply:
It seems like we are missing "method" here:

the sqlite3.Connection.create_function() -> the sqlite3.Connection.create_function() method

Reply:
And please add "Patch by Sergey Fedoseev.".